### ISSUE TITLE
Unmet peer dependency graphql

### ISSUE THREAD

**Issue by nikosantis** [neo4j-graphql/neo4j-graphql-js#436]
Hi, i have a problems with peer dependencies
Grapqhl is in v15.0.0 but

├─┬ apollo-server@2.13.1
│ └─┬apollo-server-core@2.13.1
│   └─┬ @types/graphql-upload@8.0.3
│     └── graphql@14.6.0
├── UNMET PEER DEPENDENCY graphql@15.0.0

neo4j-graphql-js@2.14.0
  └── UNMET PEER DEPENDENCY graphql@14.6.0

:(

**Comment by johnymontana** [comment]
Yes, currently we include graphql v14.2.x as a direct dependency, but plan to move it to peer dependency in the next major version release of this library which will allow usage with other versions of graphql

see #249

**Comment by michaeldgraham** [comment]
https://github.com/neo4j-graphql/neo4j-graphql-js/issues/608
